.gitignore

```diff
@@ -4,7 +4,7 @@
 *.modules/.terraform/
 *.terraform*
 *.tfstate*
-*plan.yaml
+workloads/amd64/plan.yaml
```

Comment:
should we be ignoring the plan.yaml?

Reply:
yes...since its created in each upgrade running a new one.